src/rebar3_hex_publish.erl

```diff
@@ -538,6 +559,11 @@ to_list(X)
   when erlang:is_list(X) ->
     X.
 
+
+
+help(app) ->
+    "Specifies the app to use with the publish command, currently only utilized for publish and revert operations"
```

Comment:
`--app` isn't used by publish _yet_ . I'll be doing a PR tomorrow to make it so.